nitime/tests/test_utils.py

```diff
@@ -286,11 +288,15 @@ def test_detect_lines_2dmode():
     Test multi-sequence operation
     """
 
+    # This seed affects not just the signal we generate below, but then also
+    # detect_lines->dpss_windows->tridi_inverse_iteration
+    np.random.seed(0)
```

Comment:
Hoping this will fix https://github.com/nipy/nitime/actions/runs/9485767253/job/26138519510 :crossed_fingers:

Reply:
For the sake of relaxing equality to floating-point closeness and also getting readable debug outputs, consider `git-apply`ing this patch:

```diff
diff --git a/nitime/tests/test_utils.py b/nitime/tests/test_utils.py
index c5f0a81..92daca7 100644
--- a/nitime/tests/test_utils.py
+++ b/nitime/tests/test_utils.py
@@ -299,9 +299,9 @@ def test_detect_lines_2dmode():
 
     npt.assert_(len(lines)==3, 'Detect lines failed multi-sequence mode')
 
-    consistent1 = (lines[0][0] == lines[1][0]).all() and \
-      (lines[1][0] == lines[2][0]).all()
-    consistent2 = (lines[0][1] == lines[1][1]).all() and \
-      (lines[1][1] == lines[2][1]).all()
-
-    npt.assert_(consistent1 and consistent2, 'Inconsistent results')
+    # Frequencies
+    assert np.allclose(lines[0][0], lines[1][0])
+    assert np.allclose(lines[0][0], lines[2][0])
+    # Betas
+    assert np.allclose(lines[0][1], lines[1][1])
+    assert np.allclose(lines[0][1], lines[2][1])
```

Reply:
I went with `npt.assert_allclose` because it will have nicer output on failure than `assert np.allclose(...)` but agreed either is better than what is there now :)